.github/workflows/unlocked.yml

```diff
@@ -75,6 +75,9 @@ jobs:
       - name: Test marshal regression
         run: ruby scripts/update_suite.rb -m
 
+      - name: Test incremental regression
+        run: ruby scripts/update_suite.rb -i
+
```

Comment:
It takes a second to run, so we could easily also have these in the locked workflow.

Reply:
Sure, I can do that. I guess we can also always remove it again if it takes too long when we have more incremental test cases.